apps/src/lib/levelbuilder/lesson-editor/UploadImageDialog.jsx

```diff
@@ -90,6 +102,22 @@ export default class UploadImageDialog extends React.Component {
           </div>
         )}
 
+        <label style={styles.label}>
+          Expandable
+          <input
+            type="checkbox"
+            checked={this.state.expandable}
+            style={styles.checkbox}
+            onChange={() => this.setState({expandable: !this.state.expandable})}
```

Comment:
I believe this should be something like
```suggestion
            onChange={(e) => this.setState({expandable: e.target.checked})}
```